src/propcache/__init__.py

```diff
@@ -1,6 +1,30 @@
 """propcache: An accelerated property cache for Python classes."""
 
-__version__ = "1.0.0.dev0"
+from typing import TYPE_CHECKING, List
+
+__version__ = "0.2.0.dev0"
 
 # Imports have moved to `propcache.api` in 1.0.0+.
-__all__ = ()
+# This module is now a facade for the API.
+if TYPE_CHECKING:
+    from .api import cached_property, under_cached_property
+
+__all__ = ("cached_property", "under_cached_property")
+
+
+def _import_facade(attr: str) -> object:
+    """Import the public API from the `api` module."""
+    if attr in __all__:
+        from . import api  # pylint: disable=import-outside-toplevel
+
+        return getattr(api, attr)
+    raise AttributeError(f"module 'propcache' has no attribute '{attr}'")
```

Comment:
I think `propcache` could be replaced with `{__package__}` here to avoid hardcoding the name...

Reply:
adjusted in #32